index.html

```diff
@@ -19,27 +19,46 @@
     <!-- Enables media queries on mobile -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap" rel="stylesheet">
+    <style>@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap');</style>
+
     <!-- CSS-->
     <link type="text/css" rel="stylesheet" href="css/main.css" />
+    <link type="text/css" rel="stylesheet" href="css/welcome.css" />
 
     <title>Regis Altare Booster Pack Simulator</title>
   </head>
 
   <body>
     <main>
-      <h1>The Regis Altare Birthday Project 2023</h1>
 
-      <nav>
-        <a class="active" href="index.html">Collection</a>
-        <a href="pages/message-board.html">Birthday Message Board</a>
-      </nav>
+      <section id="welcome-splash-1" class="welcome-splash">
+        <h2>Happy Birthday Altare!</h2>
+        <h3>Let's see what the Altventurers have made for you</h3>
+        <!-- TODO: change this to an svg or img of an arrow -->
+        <button onclick="toggleWelcomeTwo()">arrow</button>
+      </section>
 
-      <section id="card-list">
-        <!--Test image serving-->
-        <img class="card-image" src="images/elysium_delivery_services.jpg" />
-        <tcg-card></tcg-card>
+      <section id="welcome-splash-2" class="welcome-splash">
+        <img id="altare-img" src="images/AltareSplash.png" />
+        <nav id="welcome-nav">
+          <a href="index.html">Play Gacha</a>
+          <a href="collection.html">Full Deck</a>
+          <a href="pages/messages.html">Artist/Writer Bday Messages</a>
+          <a href="pages/message-board.html">Kudoboard</a>
+          <a href="pages/credits.html">Credits</a>
+        </nav>
       </section>
 
+      <div id="splash-overlay" class="active">
```

Comment:
(nit) loading-overlay would be a clearer name for me. All the screens having "splash" made it hard to distinguish them :')